### DISCUSSION TITLE
Option to sort by filename

### DISCUSSION THREAD

**Discussion by phi-sigsegv** [gerbera/gerbera#1308] [Ideas]
I am using gerbera with an import javascript, where i added a Directories container like for Videos and Photos. Here i want my audio file in a directory to be sorted by filename instead of by title or track number. Is that possible? If not, then this would be a nice feature, isn't it?

**Comment by phi-sigsegv** [comment]
Hm. This is more a feature request rather than a discussion. So i created issue #1311.
Feel free to delete this discussion.

**Reply by KarlStraussberger** [reply]
As by definition albums are sorted by track number and plain container by track name. In order to get the sort order changed, change the type from `UPNP_CLASS_CONTAINER_MUSIC_ALBUM `to `UPNP_CLASS_CONTAINER`
